.github/workflows/update-maintainers.yml

```
@@ -0,0 +1,134 @@
+name: Update MAINTAINERS.yaml
+
+on:
+  pull_request:
+    types: [closed]
+    paths:
+      - 'CODEOWNERS'
+
+jobs:
+  update-maintainers:
+    if: github.event.pull_request.merged
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout main branch
+        uses: actions/checkout@v2
+        with:
+          ref: master
+          path: current_state
+
+      - name: Checkout one commit before last one
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+          ref: master
+          path: previous_state
+
+      - run: cd previous_state && git checkout HEAD^
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```

```suggestion
        uses: actions/setup-node@v3
        with:
          node-version: '16'
```

always latest please